### ISSUE TITLE
Lava Rivers in Ruins.. again?

### ISSUE THREAD

**Issue by wwtall1** [yogstation13/yogstation#2388]
So, this happened on the server again, round counter says round 10519 (if it matters)

![screenshot_1](https://user-images.githubusercontent.com/21294637/28244284-796b5fbc-69ac-11e7-9867-57539ea2291d.png)

![screenshot_117](https://user-images.githubusercontent.com/21294637/28244285-7bfbeb98-69ac-11e7-8637-c9d2a4568ab5.png)

Not sure why it happened, but.. well, it did. Assuming it isnt actually part of the ruin to have lava like that.

**Comment by Qustinnus** [comment]
what ruin is this?

**Comment by Qustinnus** [comment]
I know why this is happening I think, the map has /area/lavaland/surface/outdoors which is the whitelisted area for rivers to spawn in, the walls have area/ruin/powered which is why this is not happening, area/ruin/powered or area/ruin/unpowered should be used for stuff like this

tldr; mappers git gud

**Comment by LarissaMayrink** [comment]
By the layout this looks like the abductor ship with a cosmic walker egg

**Comment by Cubetastic** [comment]
Came here to make an issue, then saw this one up.

Heres the same thing again:

http://puu.sh/wOnYN/c510ea1e89.jpg

Its iin the cosmic ashwalker ruin. "Abductor Terror Ship"

**Comment by wwtall1** [comment]
![image](https://user-images.githubusercontent.com/21294637/28933157-bc8e12a0-7841-11e7-9908-9d094e33e687.png)


it happened again, yes its the abductor lavaland ruin.
